src/color/types.ts

```diff
@@ -10,6 +10,8 @@ export interface ValidationTypes {
     Green: ValidationElementAttributes;
     Orange: ValidationElementAttributes;
     Red: ValidationElementAttributes;
+    Blue: ValidationElementAttributes;
+    DarkMode: ValidationElementAttributes;
```

Comment:
This should not be a part of the colour type. It should be another theme instead